### ISSUE TITLE
Codelabs tutorial code snippets do not match example code

### ISSUE THREAD

**Issue by chmod0775** [googlecodelabs/webrtc-web#83]
As it stands, codelab tutorial snippets do not match project code. 
For example, the tutorial at [Stream video with RTCPeerConnection](https://codelabs.developers.google.com/codelabs/webrtc-web/#4) has the following snippet:

```
function onIceCandidate(pc, event) {
  if (event.candidate) {
    getOtherPc(pc).addIceCandidate(
      new RTCIceCandidate(event.candidate)
    ).then(
      function() {
        onAddIceCandidateSuccess(pc);
      },
      function(err) {
        onAddIceCandidateError(pc, err);
      }
    );
    trace(getName(pc) + ' ICE candidate: \n' + event.candidate.candidate);
  }
}
```

However, the analogous sample code from [the github repository](https://github.com/googlecodelabs/webrtc-web) appears to have entirely different variable names. pc1 is now localPeerConnection and pc2 appears to be remotePeerConnection.

```
 if (iceCandidate) {
    const newIceCandidate = new RTCIceCandidate(iceCandidate);
    const otherPeer = getOtherPeer(peerConnection);

    otherPeer.addIceCandidate(newIceCandidate)
      .then(() => {
        handleConnectionSuccess(peerConnection);
      }).catch((error) => {
        handleConnectionFailure(peerConnection, error);
      });

    trace(`${getPeerName(peerConnection)} ICE candidate:\n` +
          `${event.candidate.candidate}.`);
  }
}

```

**Comment by nitobuendia** [comment]
@chmod0775 Thanks a lot for reporting. It feels like we updated the codelab repository to fix some issues; but we may have neglected or forgotten to do the same on the codelab text. I have worked on it for step-01 and step-02 which I presume are the ones that have the biggest differences. Have you encountered major differences in other parts too?

@samdutton I have made the suggested changes and sent it your way.

**Comment by chmod0775** [comment]
@nitobuendia Yeah, glad to help! I'll run through codelab again this week. If I find anything else, I'll be sure to comment in here.

**Comment by chmod0775** [comment]
@nitobuendia, Most of this stuff is a little pendantic, just a few typos:

[**Stream video with RTCPeerConnection**](https://codelabs.developers.google.com/codelabs/webrtc-web/#4)
1. `pc1` and `pc2` should be switched to `localPeerConnection` and `remotePeerConnection`, respectively

> In this example, the two RTCPeerConnection objects are on the same page: `pc1` and `pc2`. Not much practical use, but good for demonstrating how the APIs work.
> 

2. s/Connectionpc1/Connection/

> **Bonus points**
..
..
.. 
> 4. From the console, look at `localPeerConnectionpc1.localDescription`. What does SDP format look like?

Thank you for your work.
